sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestProxyTestBase.java

```diff
@@ -20,4 +22,29 @@ public abstract class TestProxyTestBase extends TestBase {
     public TestProxyTestBase() {
         super();
     }
+
+    private static TestProxyManager testProxyManager;
+
+    /**
+     * Before tests are executed, determines the test mode by reading the {@code AZURE_TEST_MODE} environment variable.
+     * If it is not set, {@link TestMode#PLAYBACK}
+     */
+    @BeforeAll
+    public static void setup() {
```

Comment:
CI is failing as tests extending this had a non-static `setup` or `teardown` method. Let's name these to something that won't break those SDKs such as `setupTestProxy` and `teardownTestProxy`.